### DISCUSSION TITLE
Cannot use inpaint zoom

### DISCUSSION THREAD

**Discussion by Parkergking** [richrobber2/canvas-zoom#87] [General]
I have not been able to get the inpaint zoom to work. I am using webui version 1.5.1.

**Comment by Rinkashikachi** [comment]
I cannot use zoom in controlnet. checkmark for controlnet compatibility in settings is enabled

**Reply by Parkergking** [reply]
This?

https://imgur.com/a/Io8oaov

**Comment by daswer123** [comment]
Hi, please create and write in the issue 

@Rinkashikachi Make sure you have more than 1 unit enabled in controlNet.
